modules/database/src/adapters/DatabaseAdapter.ts

```diff
@@ -105,6 +105,9 @@ export abstract class DatabaseAdapter<T extends Schema> {
   }
 
   private assignFields(schema: ConduitSchema) {
+    if (!schema.modelOptions.timestamps) {
```

Comment:
This will update explicit `false` values to `true`.
We should be checking for `undefined` || `null`.